lang/en/filter_imageopt.php

```diff
@@ -29,6 +29,8 @@
 $string['donotloadonvisible'] = 'No placeholding, load immediately';
 $string['loadonvisibilityall'] = 'All images';
 $string['loadonvisibilityafter'] = 'After {$a} image(s)';
+$string['lovsmallimage'] = 'Load when visible for small images';
+$string['lovsmallimagedesc'] = 'Apply "Load when visible" for images which are smaller than "Maximum image width"';
```

Comment:
Consider changing this to

Apply "load when visible" to all images, regardless of size